com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -732,9 +732,18 @@ public static ProbeSubdivisionResult BakeBricks(ProbeSubdivisionContext ctx)
                     {
                         var go = renderer.component.gameObject;
                         int rendererLayerMask = 1 << go.layer;
+                        renderer.volume.CalculateCenterAndSize(out _, out var rendererBoundsSize);
+                        float rendererBoundsVolume = rendererBoundsSize.x * rendererBoundsSize.y * rendererBoundsSize.z;
```

Comment:
To avoid issues with planes I'd say we should min all dimensions to avoid 0, probably to 2/3 cm?

Reply:
So, in this renderer size, there is already padding of 1cm in every axis